packages/components/src/components/chat-search/Instructions.vue

```diff
@@ -0,0 +1,108 @@
+<template>
+  <div class="instructions">
+    <h2>Navie, the AppMap AI</h2>
+    <div>
+      <p>
+        Navie is an AI software architect. You ask Navie questions about your code, and she provides
+        comprehensive answers about code architecture and design. You can also work with Navie to
+        design new features, and to improve the performance and security of your code.
+      </p>
+    </div>
+    <div>
+      <h3>How Navie Works</h3>
```

Comment:
There is so much text in the 'How Navie Works' section, the noise it generates is likely to outweigh any clarity it intends to provide. People won't read it, and they may get confused about how to proceed. Until we have a reliable way to tell them they have enough/not enough AppMap data, the simple message of 'The better your AppMaps match your question, the better Navie's answers.' Should be enough to suffice.

Reply:
I'm not totally getting the hate for this; I wanted to put on the page the information that users need to know, and we can iterate on the presentation.

Reply:
Not really 'hate' just trying to clarify the presentation.  It's your call.

Reply:
Your suggestion is to replace all the content with `The better your AppMaps match your question, the better Navie's answers`?